CHANGES.rst

```diff
@@ -244,6 +244,9 @@ astropy.units
   where numpy scalars behave differently than python ones (e.g., taking the
   square root of a negative number). [#8876]
 
+- Removed ``normed`` keyword for histogram functions in ``function_helpers``.
```

Comment:
Don't need a change-log entry - this is `affects-dev`...

Reply:
Okay. Removed.

Reply:
Oh, I missed that this is affects-dev, then the changes look even better.